### PR TITLE
M365 Lighthouse: Update tenant page

### PR DESCRIPTION
**Pull request by matsimon**
While exploring M365 Lighthouse I realized some discrepancies between the actual portal and the documentation.

What I'm not entirely certain is if my change in the description on the "Tenant Domain" is correct, but it seems so after checking a couple of tenants.

### CODE REVIEW

**Comment by SKjerland** [comment]
@matsimon - Thanks for making these updates! I need to review your changes with the feature team. Once they approve everything, I'll merge this PR as is, or make additional updates as needed and then merge. Thanks for your patience while we work on this.

File: microsoft-365/lighthouse/m365-lighthouse-tenants-page-overview.md [hunk 1]
```diff
@@ -94,15 +94,26 @@ The Tenant overview section provides information about the customer tenant from
...
+| Tenant ID |The organization's tenant ID.|
+| Lighthouse management | The management status of Lighthouse management. |
+| Delegated access | The type of delegated access to customer tenant. |
+| Permissions |	The roles assigned to you in the tenant. Roles determine which tasks you can complete for customers, and what data you can view. |
+| Total users |The number of users assigned in the tenant. You may select this number to open the Users page for that tenant.|
+| Total devices |The number of devices enrolled in the tenant. You may select this number to open the Devices page for that tenant.|
+
+#### Customer overview section
+
+The Tenant overview section provides information about the customer tenant from its Microsoft 365 account.<br><br>
```
Comment (JohanFreelancer9) [inline microsoft-365/lighthouse/m365-lighthouse-tenants-page-overview.md:107]:
```suggestion
The Customer overview section provides information about the customer tenant from its Microsoft 365 account.<br><br>
```

File: microsoft-365/lighthouse/m365-lighthouse-tenants-page-overview.md [hunk 2]
```diff
@@ -94,15 +94,26 @@ The Tenant overview section provides information about the customer tenant from
...
-| Customer domain    |The organization's domain.|
+| Customer domain    |The organization's default domain.|
 | Website    |The organization's website. You may edit this field if no data is provided.|
-| Total users    |The number of users assigned in the tenant. You may select this number to open the Users page for that tenant.|
-| Total devices|The number of devices enrolled in the tenant. You may select this number to open the Devices page for that tenant.|
 
-#### Contacts section
+#### Customer contacts section
 
 The Contacts section provides information for key contacts within the tenants you manage, such as:
```
Comment (JohanFreelancer9) [inline microsoft-365/lighthouse/m365-lighthouse-tenants-page-overview.md:118]:
```suggestion
The Customer contacts section provides information for key contacts within the tenants you manage, such as:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @matsimon, kindly commit the suggested changes. Many thanks.

**Comment by SKjerland** [comment]
@JohanFreelancer9 - I’m the owner of this article and this PR has been assigned to me. As I noted in this PR, I’m confirming the updates with the product team and will edit as needed. Thanks!

**Comment by SKjerland** [comment]
@matsimon - While confirming your updates, we discovered some additional updates that need to be made to the article. I'm going to close this PR, since we'll be making the updates on our end in our private repository. We did note the updates you requested and will be including them as part of the larger article update. Thanks again for your help ensuring our documentation in correct! It's much appreciated.

**Comment by matsimon** [comment]
Thank you @SKjerland for the update, looking forward to the update and glad things are moving forward.

**Pull request closed without merging**